src/include/mpir_op_util.h

```diff
@@ -303,13 +308,15 @@ typedef struct {
     MPIR_OP_TYPE_MACRO(MPI_DOUBLE, double, mpir_typename_double)                                                      \
     MPIR_OP_TYPE_MACRO_HAVE_FORTRAN(MPI_REAL, MPIR_FC_REAL_CTYPE, mpir_typename_real)                                 \
     MPIR_OP_TYPE_MACRO_HAVE_FORTRAN(MPI_DOUBLE_PRECISION, MPIR_FC_DOUBLE_CTYPE, mpir_typename_double_precision)       \
-    MPIR_OP_TYPE_MACRO_HAVE_LONG_DOUBLE(MPI_LONG_DOUBLE, long double, mpir_typename_long_double)                      \
-        /* The MPI Standard doesn't include these types in the floating point group for
-         * predefined operations but MPICH supports them when possible. */
+    MPIR_OP_TYPE_MACRO_HAVE_LONG_DOUBLE(MPI_LONG_DOUBLE, long double, mpir_typename_long_double)
+
```

Comment:
This mixes a whitespace change with actual functional changes. We prefer to make whitespace changes separate.

Reply:
It is deleting the trailing slash, which semantically connects the macro to the next line comment. That comment belongs to the next macro. While the comment technically is white space, it is obvious that trailing slash was there by mistake -- even though technically does no harm.

So I believe I am correcting a semantic error here rather than a pure white space change.

Reply:
Ah, sorry. You are right. I think this looks OK otherwise. I will signoff and queue it up for merge.